src/main/java/org/vietabroader/model/VASpreadsheet.java

```diff
@@ -15,8 +17,61 @@
  */
 public class VASpreadsheet {
 
+
     private String spreadsheetId;
     private Spreadsheet spreadSheet;
+    private HashMap<String,String> columnMap;
+    private HashMap<String,List<List<Object>> > contentMap;
+
+    public void init(){
+        columnMap = new HashMap<String, String>();
+        contentMap = new HashMap<String,List<List<Object>>>();
+    }
+
+
+    /**
+     * Read first column character from the sheet/spreadsheet for testing the refreshColumn method
+     * @return The Item column character in the specific row
+     * @throws IOException
+     * @throws GeneralSecurityException
+     */
+    public String readFirstColumn() throws IOException, GeneralSecurityException{
+        refreshColumn("A");
+        return columnMap.get("Item");
+    }
+
+    /**
+     * Read first key from the Item column for testing the refreshColumn method
+     * @return The first key
+     * @throws IOException
+     * @throws GeneralSecurityException
+     */
+    public String readFirstKey() throws IOException, GeneralSecurityException{
+        refreshColumn("A");
+        List<List<Object>> l = contentMap.get("Item");
+        return l.get(1).get(0).toString();
+    }
+
+
+    //note: get(a).get(b): a row, b column
+
+    /**
+     * Get the column name and content given the character of the column
+     * @param s the character of the column
+     * @throws IOException
+     * @throws GeneralSecurityException
+     */
+    public void refreshColumn(String s) throws IOException, GeneralSecurityException{
+        String sheetname = getSheetTitles().get(0);
+        String range = sheetname + "!"+s+":"+s;
```

Comment:
We are supposed to get a column, so it should be `sheetName + "!" + col + startRow + ":" + col + toRow`